src/library/Runner.java

```diff
@@ -0,0 +1,31 @@
+package library;
+
+import java.util.Collections;
+
+public class Runner {
+
+    public static void main(String[] args) {
+
+        Library library = new Library();
+
+        Book book1 = new Book(1, "Harry Potter", "fantasy");
+        Book book2 = new Book(2, "The Autumn of the Patriarch", "novel");
+        Book book3 = new Book(3, "Ludzi na baloce", "novel");
+        Book book4 = new Book(3, "Dune", "sci-fi");
+
+        library.addBook(book1);
+        library.addBook(book2);
+        library.addBook(book3);
+        library.addBook(book4);
+
+        Collections.sort(library.getLibrary());
```

Comment:
I'd suggest to add some comparators to Library class and pass them into sort() method with list of books. This way we keep multiple possibilities to sort library depending on users choise

Reply:
Many thanks for your recommendation. Done